scripts/coverage.sh

```diff
@@ -1,136 +1,66 @@
 #!/usr/bin/env bash
 #
-# Runs all tests and collects code coverage
+# Run tests and collect code coverage
 #
-# Warning: this process is a little slow
+# Run all:
+#   $ ./script/coverage.sh
 #
+# Run for specific packages
+#   $ ./script/coverage.sh -p solana-account-decoder
+#   $ ./script/coverage.sh -p solana-account-decoder -p solana-accounts-db [-p ...]
 
-if ! command -v grcov; then
-  echo "Error: grcov not found.  Try |cargo install grcov|"
-  exit 1
-fi
-
-if [[ ! "$(grcov --version)" =~ 0.8.[0-9] ]]; then
-  echo Error: Required grcov version not installed
+set -e
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
```

Comment:
nit: hmm, do we want 5th variation of `here`???

```
$ git grep -h "^here.*=" origin-anza/master  :**.sh | sort | uniq -c | sort -n
      1 here="$(dirname "${BASH_SOURCE[0]}")"
      2 here="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
     20 here="$(dirname "$0")"
     22 here=$(dirname "$0")
```

Reply:
to me, this one is the best here haha :trollface:  we don't need to worry about path anymore. the "here" is always the scripts's pwd. also, it's easy to convert this one to a relative path. if you're okay with it, let me keep it!